recipes/zeromq/cppzmq_git.oe

```diff
@@ -0,0 +1,22 @@
+DESCRIPTION = "C++ bindings for ZeroMQ"
+HOMEPAGE = "http://www.zeromq.org"
+LICENSE = "MIT"
+SRC_URI = "git://github.com/zeromq/cppzmq.git;protocol=http;${SRC_REV}"
+SRC_REV = "commit=a88bf3e0b0bc6ed5f5b25a58f8997a1dae374c8b"
+
+S = "${SRCDIR}/cppzmq"
+
+do_install () {
+    install -d ${D}${includedir}
+    install -m 0755 ${S}/zmq.hpp ${D}${includedir}
+}
+
+# There is just one file, so anything more than just one package is silly
+PACKAGES = "${PN}"
+FILES_${PN} = "${includedir}"
+# For build-time package dependency, we pull in libzmq, which is responsible
+# for pulling in libzmq-dev also
+DEPENDS_${PN} = "libzmq"
+# For run-time package dependency (ie. sdk-image RDEPENDS), we pull in
+# libzmq-dev, which is responsible for pulling in libzmq also
+RDEPENDS_${PN} = "libzmq-dev"
```

Comment:
Correct.  But this whole recipe is just one single file.  Should we really have two packages for that?